iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/compaction/execute/performer/impl/FastCompactionPerformer.java

```diff
@@ -143,18 +143,17 @@ public void perform() throws Exception {
             x -> x.definitelyNotContains(device) || !x.isDeviceAlive(device, ttl));
         sortedSourceFiles.sort(Comparator.comparingLong(x -> x.getStartTime(device)));
         if (ttl != Long.MAX_VALUE) {
-          Deletion ttlDeletion =
-              new Deletion(
+          TreeDeletionEntry ttlDeletion =
```

Comment:
Should distinguish between tree models and table models for current device.

Reply:
Done